### DISCUSSION TITLE
7.10.6.2 (Hotfix 2)

### DISCUSSION THREAD

**Discussion by JazzyJazzjazzy** [T-Troll/alienfx-tools#279] [General]
The latest release claims Direct ACPI access. I thought this program did not use this type of access. Have you decided otherwise?

Also, this release is for the benefit of AMD chips, right? Thank you.

**Comment by T-Troll** [comment]
Yes, it should not. Can you please clarify - what tool request it? I can miss the point then build the release, so i need to fix it.

No, it's not only about AMD, but about desktops - i have a bug in desktop hardware detection.

**Reply by JazzyJazzjazzy** [reply]
From the notes... "# Fan control tools with Alienfan V1 support. This set of utilities build against older V1 (Direct ACPI access) interface." This comment led me to ask the previous question regards access. 

I will stay on the previous release as I am on an Intel laptop. Thank you, my friend.

**Reply by T-Troll** [reply]
Just use different archive (`alienfx-tools.zip`)!
V1 is for some systems it works better, but i don't recommend it if V2 works (it's default now).

PS: I don't know why, but this time GitHub put V1 .zip first, not last as usual....

**Reply by JazzyJazzjazzy** [reply]
Tried new .zip. Did not have the .reg entry and did not add it, so failed to find my equipment. Is okay, "If it ain't broke, don't fix it". I will keep previous version as I see no benefit for me. Thank you.

**Reply by T-Troll** [reply]
I modify release and add some into description. It's proved common .zip are correct and use WMI access, Fan-SDK_v1.zip is correct too but using direct ACPI (this still useful for some old systems).
